pyccl/tracers.py

```diff
@@ -8,12 +8,12 @@
                          growth_factor, scale_factor_of_chi, h_over_h0)
```

Comment:
All of these imports from `background` have become methods of `cosmo`. Could you replace all occurrences in the code below? It's cleaner, and the less we import at the top, the fewer the chances we get into cyclic import trouble in the future.

Reply:
Done